src/compiler/emitter.ts

```diff
@@ -4093,6 +4167,8 @@ module ts {
                         return write("true");
                     case SyntaxKind.FalseKeyword:
                         return write("false");
+                    case SyntaxKind.AsteriskToken:

```

Comment:
This is kind of weird - this is the only case in this `switch` where we handle non-keyword tokens.